ResourceStatusSystem/Client/ResourceManagementClient.py

```diff
@@ -1,12 +1,11 @@
 """ ResourceManagementClient
 
-  Client to interact with the ResourceManagementDB.
-
+  Client to interact with the ResourceManagement service and from it with the DB.
```

Comment:
Maybe you should be a bit more specific  in the differences between `ResourceManagement.py` and `ResourceManagementClient.py` ? Even the docs are the same :-D

Reply:
Done.

Reply:
OK much clearer. And just for my education: in general, we should use `ResourceClient`, correct ? Only if we specifically want to query RSS should we use the other one ?

Reply:
Yes, `ResourceClient` is in practice the API for RSS, and normally the only object you want to use.